sdk/cognitivelanguage/ai-language-text/review/ai-language-text.api.md

```diff
@@ -406,6 +509,25 @@ export interface KeyPhraseExtractionSuccessResult extends TextAnalysisSuccessRes
     readonly keyPhrases: string[];
 }
 
+// @public
+export enum KnownClassificationType {
```

Comment:
Is this coming directly from swagger? "kind" is much preferred over "type".

Reply:
Yes, it is coming from the swagger and for this first beta, we'll use the generated names as is.

Reply:
It's better we leave feedback on the swaggers, as I've been doing but haven't had time with these past few.

@peytonfraser FYI, "kind" as a discriminator in swagger is generally preferred over "kind". Maybe too late for 2022-10-01-preview - though, perhaps not - but the next previous should consider changing this. It may come up in API review, though we don't always have time for reviewing all the models. See https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#:~:text=DO%20define%20a%20kind%20field

Reply:
That is fine, would you mind starting an email thread to track this? I will mark this as resolved for now since it is out of the PR scope.